sdk/objc/components/video_codec/RTCVideoEncoderH264.mm

```diff
@@ -54,14 +54,42 @@ - (void)frameWasEncoded : (OSStatus)status flags : (VTEncodeInfoFlags)infoFlags
 // The ratio between kVTCompressionPropertyKey_DataRateLimits and
 // kVTCompressionPropertyKey_AverageBitRate. The data rate limit is set higher
 // than the average bit rate to avoid undershooting the target.
-const float kLimitToAverageBitRateFactor = 1.5f;
+const float kLimitToAverageBitRateFactor = 10.0f;
```

Comment:
any idea why this value had to be set so high compared to what it was before? it seems that 10x average bitrate would seem quite high, no?

Reply:
Just did some tests for camera capture, and the bit rate will be close to the target value (but not exceeding the limit). the moving picture is smooth and clear, and the bit rate of the static picture can be reduced to 1/5 of the target bit rate

Reply:
The game screen may be due to complex textures, at least 4~6mbps needs to be set to achieve smooth and clear 720p/30fps, and the camera only needs 2mbps, so this PR is good for me

Reply:
@davidzhao This is for computing data rate limits and not the average bitrate.

Reply:
got it, I don't really understand this so feel free to ignore :)